python/pythonmonkey/builtin_modules/XMLHttpRequest.js

```diff
@@ -7,11 +7,41 @@
  * 
  * @copyright Copyright (c) 2023 Distributive Corp.
  */
+'use strict';
 
 const { EventTarget, Event } = require('event-target');
 const { DOMException } = require('dom-exception');
 const { URL, URLSearchParams } = require('url');
 const { request, decodeStr } = require('XMLHttpRequest-internal');
+const debug = globalThis.python.eval('__import__("pythonmonkey").bootstrap.require')('debug');
+
+debug('xhr:module')('loaded XMLHttpRequest.js module');
```

Comment:
Is this debug message useful? The `XMLHttpRequest.js` module will always be loaded through https://github.com/Distributive-Network/PythonMonkey/blob/89a90c2483781961a1dc2804c3bba37326fbb03d/python/pythonmonkey/__init__.py#L17

Reply:
You're right - and debugging module loading should be done via DEBUG=ctx:*